go/chat/teams.go

```diff
@@ -237,6 +237,24 @@ func (t *TeamsNameInfoSource) DecryptionKeys(ctx context.Context, name string, t
 	return t.makeNameInfo(ctx, team, public)
 }
 
+func (t *TeamsNameInfoSource) EphemeralEncryptionKey(ctx context.Context, tlfName string, tlfID chat1.TLFID,
+	membersType chat1.ConversationMembersType, public bool) (keybase1.TeamEk, error) {
+	teamID, err := keybase1.TeamIDFromString(tlfID.String())
+	if err != nil {
+		return keybase1.TeamEk{}, err
```

Comment:
i'd just use named return args or make it a pointer if you want to explicits return `nil`

Reply:
why do we need so many args? should we be using the `membersType` since this won't work on `KBFS` type chats?